media-sound/audacity/audacity-3.4.2-r1.ebuild

```diff
@@ -95,16 +98,20 @@ RDEPEND="dev-db/sqlite:3
 		media-libs/sratom
 		media-libs/suil
 	)
-	mpg123? ( media-sound/mpg123 )
+	mpg123? ( media-sound/mpg123-base )
 	ogg? ( media-libs/libogg )
-	opus? ( media-libs/opus )
+	opus? (
+		media-libs/opus
+		media-libs/opusfile
+	)
 	sbsms? ( media-libs/libsbsms )
 	twolame? ( media-sound/twolame )
 	vamp? ( media-libs/vamp-plugin-sdk )
 	vorbis? ( media-libs/libvorbis )
 	wavpack? ( media-sound/wavpack )
 "
 DEPEND="${RDEPEND}
+	dev-libs/rapidjson
 	test? ( <dev-cpp/catch-3:0 )"
 BDEPEND="
 	sys-devel/gettext
```

Comment:
`CMakeLists.txt` lists `NASM` as a requirement.

```
The ASM_NASM compiler identification is unknown
Didn't find assembler

CMakeLists.txt:280: include( CMakeDetermineASM_NASMCompiler )

diff -Nuar a/audacity-3.4.2-r1.ebuild b/audacity-3.4.2-r1.ebuild
--- a/audacity-3.4.2-r1.ebuild	2024-04-07 20:56:04.464000000 +0000
+++ b/audacity-3.4.2-r1.ebuild	2024-04-07 23:59:20.715888551 +0000
@@ -114,6 +114,7 @@
 	dev-libs/rapidjson
 	test? ( <dev-cpp/catch-3:0 )"
 BDEPEND="
+	dev-lang/nasm
 	sys-devel/gettext
 	virtual/pkgconfig
 "
```

Reply:
I've figured out that Yasm is fine, too.
```
# grep -i asm *
audacity-3.4.2:20240403-010706.log:-- The ASM_NASM compiler identification is NASM
audacity-3.4.2:20240403-010706.log:-- Found assembler: /usr/bin/nasm
audacity-3.4.2:20240408-012932.log:-- The ASM_NASM compiler identification is YASM
audacity-3.4.2:20240408-012932.log:-- Found assembler: /usr/bin/yasm
```

Reply:
Thanks ;)